zookeeper-server/src/main/java/org/apache/zookeeper/server/NettyServerCnxnFactory.java

```diff
@@ -451,6 +458,9 @@ public void shutdown() {
     
     @Override
     public void start() {
+        if (listenBacklog > 0) {
```

Comment:
It doesn't make any difference, but it would nice for this condition to be consistent with the NIO implementation.
e.g. `listenBacklog != -1` or something similar